keyform/models.py

```diff
@@ -44,33 +44,21 @@ class Request(models.Model):
     PAYMENT_TYPES = (
         ('ca', _('Cash')),
         ('ch', _('Check')),
+        ('na', _('Not Applicable')),
     )
 
-    bpn_validator = RegexValidator(
-        '[mM8]\d{8}',
-        "Bearpass number must start with an 'M,' 'm,' or '8,' and followed by eight digits.'")
-
-    building = models.ForeignKey(Building)
-    student_name = models.CharField(max_length=128, blank=True)
-    reason_for_request = models.CharField(max_length=2, choices=REQUEST_TYPES)
-    amt_received = models.DecimalField(
-        max_digits=7,
-        decimal_places=2,
-        default=0,
-        blank=True,
-        verbose_name=_('Amount received'),
-        validators=[
-            MinValueValidator(
-                Decimal('0.00'))])
-    payment_method = models.CharField(max_length=2, choices=PAYMENT_TYPES, null=True, blank=True)
-    charge_amount = models.DecimalField(
-        max_digits=7, decimal_places=2, validators=[
-            MinValueValidator(
-                Decimal('0.00'))])
-    staff = models.ForeignKey(settings.AUTH_USER_MODEL)
-    bpn = models.CharField(max_length=9, verbose_name=_('M-Number'), validators=[bpn_validator])
+    bpn_validator = RegexValidator('[mM8]\d{8}', _("Bearpass number must start with an 'M,' 'm,' or '8,' and followed by eight digits."))
+
+    building = models.ForeignKey(Building, help_text='')
+    student_name = models.CharField(max_length=128, blank=True, help_text='')
+    reason_for_request = models.CharField(max_length=2, choices=REQUEST_TYPES, default='dk', help_text='')
+    amt_received = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, verbose_name= _('Amount received'), validators=[MinValueValidator(Decimal('0.00'))], help_text='')
+    payment_method = models.CharField(max_length=2, choices=PAYMENT_TYPES, null=True, blank=True, default='ca', help_text='')
```

Comment:
Since we have the new "Not Applicable" option, it no longer has the need to be nullable/blankable. Also I think we should remove the default just because there's no reason for it to be selected on page load.